pkg/cmd/buildinformation/list/list.go

```diff
@@ -0,0 +1,170 @@
+package list
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/MakeNowJust/heredoc/v2"
+	"github.com/OctopusDeploy/cli/pkg/apiclient"
+	"github.com/OctopusDeploy/cli/pkg/cmd/buildinformation/shared"
+	"github.com/OctopusDeploy/cli/pkg/constants"
+	"github.com/OctopusDeploy/cli/pkg/factory"
+	"github.com/OctopusDeploy/cli/pkg/output"
+	"github.com/OctopusDeploy/cli/pkg/util/flag"
+	"github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/buildinformation"
+	"github.com/spf13/cobra"
+)
+
+const (
+	FlagLatest    = "latest"
+	FlagFilter    = "filter"
+	FlagPackageId = "package-id"
+)
+
+type ListFlags struct {
+	Latest    *flag.Flag[bool]
+	Filter    *flag.Flag[string]
+	PackageId *flag.Flag[string]
+}
+
+func NewListFlags() *ListFlags {
+	return &ListFlags{
+		Latest:    flag.New[bool](FlagLatest, false),
+		Filter:    flag.New[string](FlagFilter, false),
+		PackageId: flag.New[string](FlagPackageId, false),
+	}
+}
+
+func NewCmdList(f factory.Factory) *cobra.Command {
+	listFlags := NewListFlags()
+
+	cmd := &cobra.Command{
+		Use:   "List",
+		Short: "List build information",
+		Long:  "List build information in Octopus Deploy",
+		Example: heredoc.Docf(`
+			$ %[1]s build-information list
+			$ %[1]s build-information ls
+			$ %[1]s build-info list
+		`, constants.ExecutableName),
+		Aliases: []string{"ls"},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return listRun(cmd, f, listFlags)
+		},
+	}
+
+	flags := cmd.Flags()
+	flags.BoolVarP(&listFlags.Latest.Value, listFlags.Latest.Name, "", false, "only return the latest build information")
```

Comment:
how is latest determined here?
We have had feedback about dropping latest in the `deploy release` command. I feel like this will only create further complaints about support for latest.

Reply:
By version

Reply:
maybe it should say according to SemVer then